lib/foreman_maintain/cli.rb

```diff
@@ -20,6 +21,7 @@ class MainCommand < Base
       subcommand 'service', 'Control applicable services', ServiceCommand
       subcommand 'backup', 'Backup server', BackupCommand
       subcommand 'restore', 'Restore a backup', RestoreCommand
+      subcommand 'installation', 'Lock/Unlock installed packages', InstallationCommand
```

Comment:
Per offline discussion, suggesting renaming the command to `foreman-maintain packages lock/unlock`